src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -140,4 +141,33 @@ public void getProjectName_ReturnsProjectName() {
         // then
         assertThat(name).isEqualTo(projectName);
     }
+    
+    @Test
+	public void getTagsToExcludeFromChart_defaultConfiguration_ReturnsEmptyList() throws Exception {
```

Comment:
1st comment:
the default configuration test that the initial value is an empty list.
i think it is necessary, to ensure that by default, no tags will get filtered.

2nd comment:
does it hurt that the test have a throws clause?
1) this is the default eclipse test template
2) if any of the executed code changes to throw a checked exception, the test will not have to change.

i don't mind removing it, but i think it is better to leave it there... (for any test...)